release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -86,28 +106,24 @@ These steps take approximately one hour to complete, and should be completed imm
     - [kubernetes-release-team-shadows](https://groups.google.com/a/kubernetes.io/g/release-team-shadows)
         - ⚠️  List of members for this group is managed in git. Please create a PR against `kuberenetes/sig-release` repo to include your email under `release-team-shadows` group in [`sig-release/group.yaml`](https://github.com/kubernetes/k8s.io/blob/main/groups/sig-release/groups.yaml)
 
-- Make sure you're included in the "official" release team file, e.g: [release 1.21](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release-team.md). If not submit a PR and add yourself.
-
-Early in the release cycle, the Enhancements Lead opens an enhancement tracking spreadsheet, e.g: [the 1.21 release spreadsheet](http://bit.ly/k8s121-enhancements). Later in the release this spreadsheet will contain important information for docs:
+Early in the release cycle, the Enhancements Lead opens a github project, e.g: [the 1.26 Enhancement Tracking](https://github.com/orgs/kubernetes/projects/98/views/1). Later in the release this spreadsheet will contain important information for docs, check the Docs view [here](https://github.com/orgs/kubernetes/projects/98/views/6):
```

Comment:
It would be worth adding more detail about how to open a GitHub Project. @reylejano could you add steps here? I'm not sure how essential this is, but I believe GitHub Project should be capitalized: 

```suggestion
Early in the release cycle, the Enhancements Lead opens a GitHub Project, e.g: [the 1.26 Enhancement Tracking](https://github.com/orgs/kubernetes/projects/98/views/1). Later in the release this spreadsheet will contain important information for docs, check the Docs view [here](https://github.com/orgs/kubernetes/projects/98/views/6):
```

Reply:
> It would be worth adding more detail about how to open a GitHub Project

This is sort of under discussion https://github.com/kubernetes/sig-release/issues/2109 but probably will be done via enhancements team